### DISCUSSION TITLE
Welcome to DAPS!

### DISCUSSION THREAD

**Discussion by tomschr** [openSUSE/daps#579] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cjschroder** [comment]
Hi! I work on the SUSE Doc Team with Toms, and I think DAPS is wonderful.

**Reply by tomschr** [reply]
Thanks Carla! :heart:
